bedrock/mozorg/templates/mozorg/home/home-quantum.html

```diff
@@ -251,14 +253,16 @@ <h3>{{ _('Need help?') }}</h3>
 {% endblock %}
 
 {% block js %}
-  {% if switch('home-fundraiser-EOY2017') and l10n_has_tag('home-fundraiser-EOY2017') %}
+  {% if LANG == 'en-US' and switch('netneutrality-12-2017') %}
+    {% javascript 'home-netneutrality-12-2017' %}
+  {% elif switch('home-fundraiser-EOY2017') and l10n_has_tag('home-fundraiser-EOY2017') %}
     {% javascript 'home-fundraiser-eoy2017' %}
   {% endif %}
 {% endblock %}
 
 {% block update_notification %}
-  {# do not show the update banner if the fundraiser banner is active #}
-  {% if not switch('home-fundraiser-EOY2017') %}
+  {# do not show the update banner if the fundraiser banner (or net neutrality banner) is active #}
+  {% if not switch('home-fundraiser-EOY2017') or not switch('netneutrality-12-2017') %}
```

Comment:
I think this should be an `and` instead of an `or` - both switches need to be off for the update banner to be shown.